blocks/identity-block/utils/validate-redirect-url.js

```diff
@@ -13,6 +13,7 @@ const validateURL = (url) => {
 		return url;
 	}
 
+	sessionStorage.setItem("ArcXP_redirectUrl", "/");
```

Comment:
Hi @accbjt, why are you setting ArcXP_redirectUrl to "/". Sounds like the value we are setting here should depend on the custom fields or the page in which the user is.